### ISSUE TITLE
Use CONFIG_BROKER env var instead of CONFIG_BROKER_URL to be consistent with other projects, like Borges

### ISSUE THREAD

**Issue by ajnavarro** [src-d/rovers#168]

**Comment by erizocosmico** [comment]
Doesn't the broker come from core-retrieval? How is it possible it has different env vars?

**Comment by ajnavarro** [comment]
I think because rovers is older than core-retrieval, and we implemented all the Rabbit stuff on rovers itself before the creation of core-retrieval.

**Comment by erizocosmico** [comment]
Indeed: https://github.com/src-d/rovers/blob/master/core/config.go#L12

Then we should not only change the env var but change all that to use the actual core.

**Comment by smola** [comment]
Note that we will probably favor `CONFIG_BROKER_URI` style instead of `CONFIG_BROKER` for variables describing a service with a URI: https://github.com/src-d/guide/pull/214
